autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -24,6 +49,7 @@ const SectionSelect: React.FC<SectionSelectProps> = ({ id }): JSX.Element => {
     );
   }
 
+  let countSelected = 0;
```

Comment:
nitpick: maybe rename this to `selectedCount` or `numSelected`? `countSelected` sounds like it would be a function name to me